OpenSim/Tools/CMC_Task.cpp

```diff
@@ -940,9 +940,9 @@ updateFromXMLNode(SimTK::Xml::Element& aNode, int versionNumber)
     //setOn(_on);
     setActive(_active[0],_active[1],_active[2]);
     //setWeight(_w[0],_w[1],_w[2]);
-    setKP(_kp[0],_kp[1],_kp[2]);
-    setKV(_kv[0],_kv[1],_kv[2]);
-    setKA(_ka[0],_ka[1],_ka[2]);
+    //setKP(_kp[0],_kp[1],_kp[2]);
+    //setKV(_kv[0],_kv[1],_kv[2]);
+    //setKA(_ka[0],_ka[1],_ka[2]);
```

Comment:
As per comment in #1335, I removed all code from the function except call to base class.